CMakeLists.txt

```diff
@@ -173,12 +184,6 @@ SET(FW_SOURCES
 )
 list(TRANSFORM FW_SOURCES PREPEND ${CMAKE_CURRENT_SOURCE_DIR}/Firmware/)
 
-foreach(_FILE ${FW_SOURCES})
-    get_filename_component(_BASE ${_FILE} NAME)
-    set_property(SOURCE ${_FILE} APPEND_STRING PROPERTY COMPILE_FLAGS "-frandom-seed=${_BASE}.o")
-endforeach()
-
-
```

Comment:
Why did you remove this? Now your compiler will use random seeds all the time...